### PR TITLE
Closes #107, #132, #134 pharmaverseadam updates for pk/pd

### LINKED ISSUE #107: Explore and test PK template tests can make use of pharmaverseadam data (part II)
part I

- [x] test-listing_pkcl02.R

part II

- [x] test-listing_pkpl02.R
- [x] test-listing_pkpl04.R

part III

part IV
- [x] test-table_pkpt04.R
- [x] test-table_pkpt05.R
- [ ] test-table_pkpt06.R
- [ ] test-table_pkpt07.R

### PR DESCRIPTION
**Pull request by zdz2101**

### CODE REVIEW

File: tests/testthat/_snaps/listing_pkpl04.md [hunk 1]
```diff
@@ -3,64 +3,64 @@
...
-      AB12345-CHN-1-id-316          199.393501681307                      NA                             NA                32.1545537098299          NA                      NA             
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+                     AUC All (h*ug/ml)     AUC All (h*ug/ml)                AUC All (h*ug/ml)                Max Conc (ug/ml)       Max Conc (ug/ml)                 Max Conc (ug/ml)            
+      Subject ID    Xanomeline Low Dose   Xanomeline High Dose   Xanomeline Low Dose/Xanomeline High Dose   Xanomeline Low Dose   Xanomeline High Dose   Xanomeline Low Dose/Xanomeline High Dose
+      ———————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————————
+      01-701-1028           NA              18.1210409513286                        NA                              NA              1.77185469787668                        NA
```
Comment (edelarua) [inline tests/testthat/_snaps/listing_pkpl04.md:16]: Are the 4th and 7th columns expected to be all NA here?

**Review by edelarua — CHANGES_REQUESTED** [review]
Looks good!! I'm not familiar with the specs for each table (i.e. if it's ok to drop the columns you did) but I'm sure you know better than I do. Just a few comments/suggestions from me :)

File: tests/testthat/_snaps/listing_pkpl04.md (hunk 1 above)
Comment (zdz2101) [reply tests/testthat/_snaps/listing_pkpl04.md:16]: @edelarua it's actually kinda how the old snapshot looked too, the "issue" with this dataset is that its the same compound for both arms just in different doses, vs the previous is was two different compounds

**Comment by zdz2101** [comment]
> Looks good!! I'm not familiar with the specs for each table (i.e. if it's ok to drop the columns you did) but I'm sure you know better than I do. Just a few comments/suggestions from me :)

I didn't really drop columns as much as it was the dataset isn't as "rich" as the previous, we only have 1 analyte where the doses are just different in the pharamverseadam data so there's no "Drug Y" equivalent to look at 

@shajoezhu do you have thoughts how this has been handled?

**Review by edelarua — APPROVED** [review]
Lgtm! Thanks Zelos

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @zdz2101 dont worry for drug y for now

**Pull request merged**